crypto/pkcs12/p12_crt.c

```diff
@@ -58,8 +60,12 @@ PKCS12 *PKCS12_create(const char *pass, const char *name, EVP_PKEY *pkey, X509 *
     if (pkey && cert) {
         if (!X509_check_private_key(cert, pkey))
             return NULL;
-        if (!X509_digest(cert, EVP_sha1(), keyid, &keyidlen))
+        md = EVP_MD_fetch(ctx, "SHA1", propq);
```

Comment:
This change probably wasn't really necessary since the X509_digest does a fetch internally anyway..